src/gflownet/algo/trajectory_balance.py

```diff
@@ -159,7 +159,7 @@ def not_done(lst):
                 fwd_logprob[i].append(log_probs[j].unsqueeze(0))
                 data[i]['traj'].append((graphs[i], graph_actions[j]))
                 # Check if we're done
-                if graph_actions[j].action is GraphActionType.Stop or t == self.max_len - 1:
+                if graph_actions[j].action is GraphActionType.Stop or (self.max_len and t == self.max_len - 1):
```

Comment:
I'm not going to prevent this, but checks like this usually mean `max_len` is not being handled well. My guess is that it is an optional parameter without a default. Could the default be some huge number instead?

Reply:
Yes that would make sense as well.

Reply:
Resolving this for now, I opened #26 as a reminder.